outdoor_dwa.py

```diff
@@ -824,6 +890,7 @@ def main():
     subLaser = rospy.Subscriber("/scan", LaserScan, obs.assignObs, config)
     subGoal = rospy.Subscriber('/target/position', Twist, config.target_callback)
     subImage = rospy.Subscriber(config.image_topic_name, Image, config.img_callback)
+    test_case_sub = rospy.Subscriber('/test_case_info',String, config.test_case_cb)
```

Comment:
Do we need this Subscriber?

Reply:
Yes this is the topic published by the manager script to signal the new goal location.

Reply:
the new goal should be published directly to /target/position.